src/app/_mermaid_common/forms/directives/numberinput.dir.js

```diff
@@ -87,7 +87,7 @@ angular.module('mermaid.forms').directive('numberinput', [
 
         // Attempt to cast model val as number for view and validators
         modelCtrl.$formatters.push(function(modelValue) {
-          if (scope.ngModel != null) {
+          if (scope.ngModel != null && modelValue[attrs.widgetName] != null) {
```

Comment:
Will this cover `typeof modelValue[attrs.widgetName] === 'undefined'`?

Reply:
@saanobhaai Are you thinking `"undefined"` or `undefined`?

Reply:
er right `undefined` sans quotes

Reply:
the `!=` should capture undefined and null